protocols/virtualdefund/helpers_test.go

```diff
@@ -0,0 +1,319 @@
+package virtualdefund
+
+import (
+	"bytes"
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/statechannels/go-nitro/channel/consensus_channel"
+	"github.com/statechannels/go-nitro/channel/state"
+	"github.com/statechannels/go-nitro/channel/state/outcome"
+	"github.com/statechannels/go-nitro/internal/testactors"
+	ta "github.com/statechannels/go-nitro/internal/testactors"
+	"github.com/statechannels/go-nitro/internal/testhelpers"
+	. "github.com/statechannels/go-nitro/internal/testhelpers"
+	"github.com/statechannels/go-nitro/protocols"
+	"github.com/statechannels/go-nitro/types"
+)
+
+// assertSideEffectsContainsMessageWith fails the test instantly if the supplied side effects does not contain a message for the supplied actor with the supplied expected signed state.
+// TODO: This is copied from https://github.com/statechannels/go-nitro/blob/0722a1127241583944f32efa0638012f64b96bf0/protocols/virtualfund/virtualfund_single_hop_test.go#L409
+// We should probably establish some shared helpers
```

Comment:
I think something is off with the godoc here (mismatch in function name).

Is it possible to avoid adding all 300 lines of test helpers by reusing anything in `internal/`?

Reply:
I think we could refactor a bunch of these helpers to be shared between `virtualFund` and `virtualDefund`. I've also moved some of the assertion helpers to `internal` in https://github.com/statechannels/go-nitro/pull/540/commits/ee43f127973b396d63a270888056cde676a3e652 so they can be used by other tests.